ui/src/app/utils/runtime-utils.ts

```diff
@@ -0,0 +1,93 @@
+import {runtimeApi} from 'app/services/swagger-fetch-clients';
+import {useOnMount} from 'app/utils';
+import {switchCase} from 'app/utils';
+import {
+  LeoRuntimeInitializer,
+} from 'app/utils/leo-runtime-initializer';
+import {
+  runtimeStore,
+  useStore
+} from 'app/utils/stores';
+import {Runtime, RuntimeStatus} from 'generated/fetch';
+import * as fp from 'lodash/fp';
+
+import * as React from 'react';
+
+const {useState, useEffect} = React;
+
+export type RuntimeStatusRequest = 'Delete';
+
+export const RuntimeStateRequest = {
```

Comment:
I'm confused as to why these two values are needed - can you just have a single enum?